python-package/lightgbm/sklearn.py

```diff
@@ -374,6 +377,10 @@ def __call__(
         See Callbacks in Python API for more information.
     init_model : str, pathlib.Path, Booster, LGBMModel or None, optional (default=None)
         Filename of LightGBM model, Booster instance or LGBMModel instance used for continue training.
+    eval_X : {X_shape}, or tuple of such inputs, or None, optional (default=None)
```

Comment:
_(changed by me in https://github.com/microsoft/LightGBM/pull/6857/commits/d7e0fff5cecc22e95027d43389667d857bdb48a3)_

Slightly modified the wording here... I thought just "tuple" was not clear when combined with the other content of `X_shape`.

Reply:
Maybe for another PR: I think the docstring of `eval_sample_weight` should be aligned with the one of `eval_y`.